ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -160,15 +160,27 @@ def prepare_host(host):
         host.put_file_contents(env_filename, env_to_script(host.to_env()))
 
 
-def allow_sync_ptr(host):
-    kinit_admin(host)
-    host.run_command(["ipa", "dnsconfig-mod", "--allow-sync-ptr=true"],
-                     raiseonerr=False)
+def rpcbind_kadmin_workaround(host):
+    """Restart rpcbind in case it blocks 749/UDP
+
+    See https://pagure.io/freeipa/issue/7769
+    See https://bugzilla.redhat.com/show_bug.cgi?id=1592883
+    """
+    result = host.run_command(['ss', '-ulnp', 'sport', '=', '749'])
+    if 'rpcbind' in result.stdout_text:
+        host.run_command(['systemctl', 'restart', 'rpcbind.service'])
```

Comment:
Is a restart guaranteed not to rebind to port 749? Should this be in a loop?

Reply:
In my experiments, rpcbind always incremented the port number by a small, random value. Nevertheless a loop doesn't hurt.

Reply:
I think it would be good to use the wait option to make sure the the restart is finished before trying another restart.

Reply:
But I am not sure that it is available in old systemd versions.